### PR TITLE
December newsletter

### PR DESCRIPTION
**Pull request by bethaitman**
Newsletter for December 2020. I messed up the image upload and it's late - I'll get rid of the duplicate tomorrow.

### CODE REVIEW

File: docs/blog/newsletter-december-2020.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+.. post:: December 03, 2020
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – December 2020
+#########################################
+
+Hey everyone - it's Beth bringing you the final WTD newsletter of the year.
+
+I hope you're all holding up okay. I haven't got any grand thoughts for you on how to sum up this year, or how we'll handle what's next. But I do want to thank all of you for being part of this community - for your conversations, your contributions, your insights and support and wit and sarcasm. They're what Write the Docs is built on. So thanks, for making our Slack and the three conferences into a source of support and connection - in a year when we've needed it more than ever.
```
Comment (plaindocs) [inline docs/blog/newsletter-december-2020.rst]: Slight preference for "how to sum up this year for you" over "for you on how to sum up this year" but YMMV

Comment (plaindocs) [reply docs/blog/newsletter-december-2020.rst]: The rest is <3 <3 <3

File: docs/blog/newsletter-december-2020.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+We're already looking towards 2021, and have `just announced </conf/portland/2021/news/welcome/>`__ that Write the Docs Portland will be our fourth virtual conference, on April 25-27. Mark your calendars, and keep an eye out for the Call for Proposals - it's coming sometime in December.
```
Comment (plaindocs) [inline docs/blog/newsletter-december-2020.rst:13]: I'd best get a move on. :-p

File: docs/blog/newsletter-december-2020.rst [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+That's it from me. I hope you enjoy our write-ups of what's been happening in Slack; and I wish you a restful holiday season, and the best of luck for 2021.
+
+-----------------------------------------------------
+Trying to change when "we've always done it that way"
```
Comment (plaindocs) [inline docs/blog/newsletter-december-2020.rst:18]: Reads well.

**Review by plaindocs — APPROVED** [review]
LGTM

File: docs/blog/newsletter-december-2020.rst (hunk 1 above)
Comment (hillaryfraley) [reply docs/blog/newsletter-december-2020.rst]: I agree, I like it a lot. After the year we've all had I think a little extra <3 is welcome.

File: docs/blog/newsletter-december-2020.rst (hunk 3 above)
Comment (hillaryfraley) [reply docs/blog/newsletter-december-2020.rst:18]: It's great! Thank you Beth.

File: docs/blog/newsletter-december-2020.rst [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+------------------------------
+The best time to use a tooltip
+------------------------------
+
+Tooltips and online help are very different tools for providing information to users. How do you choose between them?
+
+The main response from Slack was that it's not a binary choice, and both are useful when implemented well. In fact, they support each other: the brief details in a tooltip can link to online help for fuller details.
+
+Tooltips are simpler for users to access than online help, but they're limited in the amount they can contain. They're at their best when they enhance the interface - for example, showing that a component has a keyboard shortcut. Just be careful, as it's not always easy for users to tell that a tooltip is available.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-december-2020.rst]: limited in the amount of information they can contain? amount of text?

File: docs/blog/newsletter-december-2020.rst (hunk 1 above)
Comment (bethaitman) [reply docs/blog/newsletter-december-2020.rst]: Changed to Sam's suggestion, and thank you!

File: docs/blog/newsletter-december-2020.rst (hunk 3 above)
Comment (bethaitman) [reply docs/blog/newsletter-december-2020.rst:18]: Great!

File: docs/blog/newsletter-december-2020.rst (hunk 4 above)
Comment (bethaitman) [reply docs/blog/newsletter-december-2020.rst]: Changed!

**Pull request merged**